### DISCUSSION TITLE
Embed Budibase app into existing website with iframe?

### DISCUSSION THREAD

**Discussion by scott-coates** [Budibase/budibase#3936] [Ideas]
I see that some other low-code tools like AppSmith support embedding via a shareable iframe. See https://docs.appsmith.com/how-to-guides/embed-appsmith-into-existing-application.

Is this something Budibase will ever do?

**Comment by scott-coates** [comment]
Answer: https://github.com/Budibase/budibase/discussions/3890

**Reply by kaushalendra-pandey** [reply]
Hey, did you find any answer ti this?
My Budibase dashboard is public and still I am not able to embed it in my existing website. It throws `'X-Frame-Options' to 'sameorigin'.`

**Reply by dosandco** [reply]
Did anyone ever solve this?  I'd love to be able to embed a 'public' page from my Budibase app on another site.

**Comment by oswaldB** [comment]
Hi,

It would be awesome !! 

Because now the only way is to change the nginx configs and to do it at every updates...

**Comment by zhengtulymGh** [comment]
Try ```Embed```  component
<img width="1456" alt="20221017-171917" src="https://user-images.githubusercontent.com/13768025/196140355-042dfcd4-3027-44f5-a2e4-175952982d8b.png">

**Reply by scil** [reply]
that's embedding  existing website into budibase, not embedding  budibase into existing website.

**Reply by nunorosa9** [reply]
![image](https://user-images.githubusercontent.com/69484935/224494485-f935e634-41f3-437a-9d51-387f0e76309d.png)

Does the iframe integration work with the jwt library for token validation?
